Support Apache Pulsar up to version 2.6

jarcec opened the pull request.

Commit:
Update Dockerfile with new limitations on downloading JDK 8

With JDK8 moving from main support line, Oracle insist on using Oracle
account to download the bits. Using the wget with special header is no
longer enough. Thus I've updated the docker file to use local copy of
the jdk and it's assumed that someone will download it manually upfront.
Far from ideal state, but wasn't able to make it work in fully
"non-manual" way.

Another small change is just exposing PULSAR_VERSION so that I don't
have to copy&paste long URLs all the time :-)

Commit:
Support changes in pulsar-admin command

Pulsar-admin output changed mid-2.x line and is now outputting
cluster names with quotes around - so I had to update the validating
condition to take that into account and optionally remove them.

dimaspivak commented on start.py:
```diff
@@ -290,7 +290,11 @@ def main(args):
         command_status = node.execute(command, quiet=True)
-        return command_status.exit_code == 0 and command_status.output.strip() == cluster_name
+        # At some point Pulsar changed output format to enclose cluster name with quotes, so we are removing them if they are there
+        listed_cluster_name = command_status.output.strip()
```
`str.strip`, if no args are given, will strip for whitespace. If args (a string) is given, those characters will be stripped from the start and end of the string. Perhaps we might wanna do `command_status.output.strip(' "')` to remove any spaces and the double quotes in one command?

jarcec replied:
Thank you for the quick python lesson :)

dimaspivak reviewed (changes requested):
Looks great overall. Just one comment that could simplify things a tad.

Commit:
Use strip with arguments instead of custom slice

Incorporating Dima's feedback. The command does return new line
character and other white space characters, so I used two strip()
calls in sequence.
